config/config-example.js

```diff
@@ -228,6 +228,11 @@ exports.repl = true;
 exports.replsocketprefix = './logs/repl/';
 exports.replsocketmode = 0o600;
 
+// disablehotpatchall - disables allowing hotpatching every hotpatchable part
+// of PS at once. Generally speaking, there probably won't be a need to for this,
+// which is why there's an option to disable this.
+exports.disablehotpatchall = false;
```

Comment:
I wonder if we should have it in `config-example` at all? I think most servers will want to be able to `/hotpatch all`.